graphql/util.go

```diff
@@ -383,3 +383,33 @@ func ConvertDBTasksToGqlTasks(tasks []task.Task, baseTaskStatuses BaseTaskStatus
 	}
 	return taskResults
 }
+
+func isTaskAborted(at *restModel.APITask) bool {
+	if at.DispatchTime == nil {
```

Comment:
I realize this is a copy of what was there already, but I think this can simply check the aborted field on the task. In fact I think the existing code is too strict in what it does